index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+    <script src="https://kit.fontawesome.com/59ba0ac1be.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <a href="./index.html">
+    <img src="./images/logo.png" alt="Company logo" title="company logo" class="img_class">
+    </a>
+    <nav>
+      <ul>
+        <li><a href="./portfolio.html">Portfolio</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="./contact.html">Contact</a></li>
+      </ul>
+    </nav>
+    <i class="fa-sharp fa-solid fa-bars"></i>
+  </header>
+  <section id="introduction">
+    <h1 class="bigtext">I'm Damilare <br> Happy to see you today!</h1>
+    <p>I'm a software developer! I can help you build a product feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me</p>
+    <h3 class="connect">LET'S CONNECT</h3>
+    <span>
+        <a href="http://twitter.com/Dammybest"><i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="https://www.linkedin.com/in/damilare-ismaila-4a5a8b30/"><i class="fa-brands fa-linkedin-in"></i>
+        </a>
+        <a href="http://github.com/Bestbynature"><i class="fa-brands fa-github"></i>
+        </a>
+        <a href="http://facebook.com/bestbynature"><i class="fa-brands fa-facebook"></i>
+        </a>
+    </span>
```

Comment:
- [ ] Kindly consider taking a `ul` instead of `span` and in the `ul` you can take `li`, and then `a` tag inside each `li` for each icon. this is for the best practices and for the best semantics.